src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +16,28 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+
+    static String FORMATER = "w";
```

Comment:
constants should have `public` and `final` modifiers
```suggestion
    public static final String FIRST_CHARACTER = "w";
```